lib/salus/scanners/report_go_dep.rb

```diff
@@ -1,44 +1,105 @@
 require 'toml'
 require 'salus/scanners/base'
+require 'salus/plugin_manager'
+require 'salus/report'
 
 # Report the use of Go packages captured in a Gopkg.lock files.
 # https://github.com/golang/dep
 
 module Salus::Scanners
   class ReportGoDep < Base
     def run
-      unless should_run?
```

Comment:
any reason the "unless should_run?" is being dropped?

Reply:
I was just a little worried it might be redundant w/the conditionals below covering the same ground - added it back!